mate-base/mate-session-manager/mate-session-manager-1.14.0.ebuild

```diff
@@ -0,0 +1,104 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+
+GCONF_DEBUG="yes"
+
+inherit autotools gnome2 versionator
```

Comment:
you can't inherit autotools, it does not exist. 
Please use repoman to validate your ebuilds.